### ISSUE TITLE
LivePaper Error

### ISSUE THREAD

**Issue by raaji9** [zuranthus/LivePaper#14]
![{C6844DCE-7051-4FCC-9DBE-1C0AFD22C887}](https://github.com/user-attachments/assets/561ceeb3-3610-4144-8949-2a70bc2d0c79)

Regardless of the file type chosen, be it MP4, GIF, or WEBP, none seem to function properly. Are there any potential solutions?

**Comment by thechief77** [comment]
Same by me. Did you find the solution?

**Comment by raaji9** [comment]
No, I think it's because of the new version of windows 11 24H2. I hope it gets fixed asap 😕.

**Comment by thechief77** [comment]
How it's resource usage by the way? I never used it but i want to because of it's lightweight being (it doesn't work properly tho). As you used it for a while could you share some info about resource usage?

**Comment by zuranthus** [comment]
Hey, thanks for reporting!

I've attempted to fix the issue. If you want to try it out, download the latest builds https://github.com/zuranthus/LivePaper/releases/tag/latest

**Comment by raaji9** [comment]
![image](https://github.com/user-attachments/assets/56d70f66-6b71-43da-b402-e2c207163748)
The app is just running in the background, and nothing is happening (´･ω･`)?

**Comment by raaji9** [comment]
![Uploading image.png…]()
The app is just running in the background, and nothing is happening (´･ω･`)?

**Comment by raaji9** [comment]
![image](https://github.com/user-attachments/assets/1f70f141-5fc9-47ba-a228-242f9835c085)
Now the app is just running in the background, and nothing is happening (´･ω･`)?
I wish you good luck for fixing the issue and thank you for your time and effort on this project <3

P.S. Sorry about the closing and reopening the issue, I didn't mean to ಥ_ಥ

**Comment by raaji9** [comment]
> How it's resource usage by the way? I never used it but i want to because of it's lightweight being (it doesn't work properly tho). As you used it for a while could you share some info about resource usage?

I've played different types of games, some take low resources while some take a lot and in all those times, I've never felt any drop in frames or slower system performance. Even doing your casual work or leaving the screen just like that doesn't affect much. Also, sorry because I can't provide you any resource usage statistics as I've resettled my laptop a lot, but I can assure you that It's the best one I've used so far.
